test/generators_test.jl

```diff
@@ -128,4 +128,50 @@ using LinearAlgebra
         @test nnz(Asub) == 5*4
 
     end
+
+    @testset "partial duplication" begin 
+        
+        A = sprand(100,100,.2)
+        A = max.(A,A')
+        B = MatrixNetwork(A)
+
+        @testset "get row" begin 
```

Comment:
Either remove this test or make this a callable function in src. At the moment, this test is decoupled from the code in /src and should be removed.